src/qt/test/test_main.cpp

```diff
@@ -46,7 +45,7 @@ int main(int argc, char *argv[])
 
     // Don't remove this, it's needed to access
     // QCoreApplication:: in the tests
```

Comment:
It's still accurate, but I expanded the comment in f4d28fbe1b8041ce358d80ecb0e1fa074eee359c.